src/__test__/__snapshots__/Missions.test.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Render Missions component correctly Render missions component with actions correctly 1`] = `
+<div>
+  <div
+    class="table-container"
+  >
+    <table
+      class="mission-table"
+    >
+      <thead>
+        <tr>
+          <th>
+            Mission
+          </th>
+          <th>
+            Description
+          </th>
+          <th
+            class="status"
+          >
```

Comment:
- [ ] 👋 Hey team! You're doing a great job with your React testing library for component rendering. 🧪🚀 However, for a more comprehensive test coverage, it'd be fantastic if we could also include unit tests for our pure functions, especially in the Redux part of our codebase. This ensures that our logic is solid and dependable. 🧐 Let's aim for that well-rounded test suite! Thanks for your dedication to quality code.